cockroachcloud/ccloud-get-started.md

```diff
@@ -317,6 +326,35 @@ warning: server version older than client! proceed with caution; some features m
 user@free-tier7.gcp-us-central1.crdb.io:26257/defaultdb>
 ~~~
 
+### Connect to your cluster using SSO
+
+Use the `--sso` flag to connect to your cluster using [single sign-on (SSO) authentication](cloud-sso-sql.html). 
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql --sso dim-dog
+~~~
+
+This will open a browser window on the local machine where you will login to your organization if you are not already authenticated.
+
+If you are running `ccloud` on a remote machine, use the `ccloud cluster sql -sso --no-redirect` flag, or type "n" when prompted by `ccloud` to redirect to your browser. `ccloud` will output a URL that you must copy and paste in your local machine's browser in order to authenticate. After authentication, paste in the authentication code you received in the remote terminal to complete the login process.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql --sso --no-redirect dim-dog
+~~~
+
+Using SSO login requires that a separate SSO SQL user for your account is created on each cluster. SSO SQL usernames are prefixed with `sso_`. If you haven't already created an SSO SQL user on your cluster, you will get an error. The error message will include the command to create the SSO SQL user, which you can run if you have `admin` privileges.
```

Comment:
* If the SSO user does not exist, they will first need to create a SSO SQL user with a specific username that was generated for them by CC Console earlier on. The user must **learn what this username is by going through this flow** (they should get an error back that contains the username) so that a user with that particular username is created. If they do not create a user with that particular auto-generated  SSO SQL username, the access token will not be accepted. ccloud CLI will also provide a suggested create sql user command that contains the correct username. 
* nit: "the cluster you are trying to connect to" instead of "on each cluster"
* The user also might get an error if they are logged into the wrong org. We should suggest that they run `ccloud auth logout` and then `ccloud auth login --org <organization label` to ensure they are switched into the correct org

Reply:
Done.